spec/vrt_spec.rb

```diff
@@ -32,6 +32,12 @@
     end
   end
 
+  describe '#get_map' do
```

Comment:
can you add something that basically verifies the memoization? i.e. when it's called again the method internals are not re-run